fix (api): pan correction

vcoindet opened the pull request:
## Description
Small PR to correct the pan function in GlobeControls.
changed element.clientHeight to element.clientWidth for the left pan.
<!--- Describe your changes in detail -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
<!--- Please also state your testing environment (browser, version and anything relevant) here -->

## Screenshots (if appropriate)

autra commented on src/Renderer/ThreeExtended/GlobeControls.js:
```diff
@@ -518,7 +518,7 @@ function GlobeControls(view, target, radius, options = {}) {
 
             // we actually don't use screenWidth, since perspective camera is fixed to screen height
-            this.panLeft(2 * deltaX * targetDistance / element.clientHeight);
+            this.panLeft(2 * deltaX * targetDistance / element.clientWidth);
```
actually I'm not sure about that. `targetDistance` is calculated from the camera fov, which is the *vertical* opening of the camera. Hence we normalize with `element.clientHeight`. 

For this reason (I believe), according to my tests, the mouse follow the ground better with the old code when you do some left-right pan. Thoughts?

autra replied:
To precise things after our irc discussion, what I've said above is only true because `camera.aspect` is correctly set to `width / height`. I guess this won't work if someone wants to play with a different aspect (have a skewed scene). The correct calculus should then be:

```js
this.panLeft(2 * deltaX * targetDistance * camera.aspect / element.clientWidth);
```

(sanity check, if camera.aspect is `element.clientWidth / element.clientHeight`, then it's the same as `2 * deltaX * targetDistance / element.clientHeight`).

Do you want to amend your PR with this calculus?

vcoindet replied:
OK, I changed the file as you suggested.
Note that it doesn't change anything in our case right now, but it will work correctly if one allows to change the ratio `width / height`

vcoindet committed:
fix (api): pan correction

small PR so there won't be problems if one choose to allow to change the ratio width / height.

autra commented:
Thanks!
